README.rst

```diff
@@ -35,3 +35,27 @@ library contains the following high-level features:
    :target: https://travis-ci.org/canonical-ols/acceptable
 .. |Coverage Status| image:: https://coveralls.io/repos/github/canonical-ols/acceptable/badge.svg?branch=master
    :target: https://coveralls.io/github/canonical-ols/acceptable?branch=master
+
+
+Documentation (beta)
+--------------------
+
+One of the goals of acceptable to use the metadata about your api to build documentation.
```

Comment:
I think you're missing an 'is' here? 

"One of the goals of acceptable _is_ to use the metadata about your api to build documentation."